golang/server/server.go

```diff
@@ -114,39 +117,69 @@ func (server MinimalGRPCServer) RunUntilStopped(stopper <-chan struct{}) error {
 	}
 
 	grpcServerResultChan := make(chan error)
+	mux := cmux.New(listener)
+	grpcWebL := mux.Match(cmux.HTTP1Fast())
```

Comment:
Just Curious - Whats `L` here? Can we do HTTP2?
More curiosity - How does this compare to exposing HTTP on a different Port?

Reply:
L means listener here; so we have two different listeners on the same port. Talked offline; for now will keep it and work on either having two separate ports or utilize connect-go by buf.

Reply:
can we call it Listener instead?